cachito/workers/pkg_managers/gomod.py

```diff
@@ -302,16 +303,16 @@ def resolve_gomod(app_source_path, request, dep_replacements=None, git_dir_path=
 
         if not should_vendor:
             # Make Go ignore the vendor dir even if there is one
-            go_list = ["go", "list", "-mod", "readonly"]
+            go_list = ["go", "list", "-e", "-mod", "readonly"]
         else:
-            go_list = ["go", "list"]
+            go_list = ["go", "list", "-e"]
```

Comment:
now that I'm looking at it, maybe we should define the `go_list` variable above the first usage of `go list` and use it consistently everywhere

We'll avoid more inconsistencies with passing -mod=readonly only sometimes

Reply:
Yeah, that makes sense :+1: